packages/react-storage/src/components/StorageBrowser/createStorageBrowser/types.ts

```diff
@@ -3,7 +3,6 @@ import React from 'react';
 import {
   CustomActionConfigs,
   DefaultActionConfigs,
-  ExtendedActionConfigs,
```

Comment:
Do we use `ExtendedActionConfigs` elsewhere?

Reply:
Yep. they are used in actions. It makes sense since actions assumes resolved Action configs